flyspell-correct.el

```diff
@@ -56,6 +56,36 @@ misspelled word. It has to return either replacement word
 or (command, word) tuple that will be passed to
 `flyspell-do-correct'.")
 
+(defvar  flyspell-direction t
+  "Used by function `flyspell-popup-wrapper' to determine and toggle direction of search, `t' being reverse and `nil' being forward, in order to retain backward compatability with the prior operation of `flyspell-correct'.")
+
+
+;; Convenience wrapper function for most uses
+
+;;;###autoload
+(defun flyspell-popup-wrapper (arg)
```

Comment:
`flyspell-correct.el` is an interface agnostic module. You should not mention `popup` here.